recipe/meta.yaml

```diff
@@ -1,41 +1,37 @@
 {% set name = "healpix_cxx" %}
-{% set version = "3.31" %}
-{% set sha256 = "ddf437442b6d5ae7d75c9afaafc4ec43921f903c976e25db3c5ed5185a181542" %}
+{% set version = "3.80" %}
+{% set sha256 = "8de4aabaa3d55e0c63a62c621bfd77298732c6b75782016217f5659143acbcf5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: Healpix_{{ version }}_2016Aug26.tar.gz
-  url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2016Aug26.tar.gz
+  fn: Healpix_{{ version }}_2021Jun22.tar.gz
+  url: https://downloads.sourceforge.net/project/healpix/Healpix_{{ version }}/Healpix_{{ version }}_2021Jun22.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1003
+  number: 0
   skip: true  # [win]
 
 requirements:
   build:
-    - autoconf
-    - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make
   host:
     - cfitsio
   run:
     - cfitsio
 
 test:
   commands:
-    - test -f ${PREFIX}/lib/libhealpix_cxx.a
-    - test -f ${PREFIX}/lib/libc_utils.a
-    - test -f ${PREFIX}/lib/libcxxsupport.a
-    - test -f ${PREFIX}/lib/libfftpack.a
-    - test -f ${PREFIX}/lib/libsharp.a
+    - test -f ${PREFIX}/lib/libhealpix_cxx.so  # [linux]
+    - test -f ${PREFIX}/lib/libsharp.so  # [linux]
+    - test -f ${PREFIX}/lib/libhealpix_cxx.dylib  # [osx]
+    - test -f ${PREFIX}/lib/libsharp.dylib  # [osx]
```

Comment:
I'm confused. I thought libsharp was statically linked here?

Reply:
It is dynamically linked here, I think the confusion must come from the fact that in the https://github.com/conda-forge/libsharp-feedstock it is statically linked. Please see https://github.com/conda-forge/healpix_cxx-feedstock/pull/12#discussion_r764954041 above.

Reply:
Ahhhhhh ok. We really should be shipping the libsharp dynamic libraries in the libsharp feedstock. Looking now, I see that this would involve having to install the libsharp python package at run time as well. :/

Reply:
Is there a way to compile libsharp into the source so no libsharp binaries, either static or dynamic, are in this build?

Reply:
I have been able to combine both libsharp and libhealpix_cxx static libraries into a single shared library using the following command:
```
${CC} -shared -o $PREFIX/lib/libhealpix_cxx.so -Wl,--whole-archive $PREFIX/lib/libsharp.a $PREFIX/lib/libhealpix_cxx.a -Wl,--no-whole-archive
rm $PREFIX/lib/libsharp.a
rm $PREFIX/lib/libhealpix_cxx.a
```
Does this look okay?

Reply:
You should make sure to use the input cflgss and ldflags in the environment.

Reply:
Sorry for the delay, but I have added cxxflags and ldflags to the creation of the combined `libhealoix_cxx`